envconfig.go

```diff
@@ -74,8 +78,23 @@ func Process(prefix string, spec interface{}) error {
 			f = f.Elem()
 		}
 
-		alt := ftype.Tag.Get("envconfig")
+		// Default to the field name as the env var name (will be upcased)
 		fieldName := ftype.Name
+
+		// Best effort to un-pick camel casing as separate words
+		if ftype.Tag.Get("multi_word") == "true" {
+			words := expr.FindAllStringSubmatch(ftype.Name, -1)
+			if len(words) > 0 {
+				var name []string
+				for _, words := range words {
+					name = append(name, strings.ToUpper(words[0]))
+				}
+
+				fieldName = strings.Join(name, "_")
```

Comment:
Hmm, you're right, but that means there's also an existing bug here: https://github.com/relistan/envconfig/blob/8452a1643343764c9478f2efb16b84edd75b1ca2/envconfig.go#L99

I'll fix both and re-push.